tests/v2/validation/pipeline/Jenkinsfile.multibranch.recurring

```diff
@@ -0,0 +1,118 @@
+#!groovy
+node {
+    def rootPath = "/root/go/src/github.com/rancher/rancher/"
+    def workPath = "/root/go/src/github.com/rancher/rancher/tests/v2/validation/"
+    def rancher_image_tags = "${env.RANCHER_IMAGE_TAGS}".split(',')
+    def jobName = "${JOB_NAME}"
+    def config = env.CONFIG
+    def filename = "config.yaml"
+    def releaseBranchTagMap = ["v2.7-head":"release/v2.7", "v2.8-head":"release/v2.8", "v2.9-head":"release/v2.9"]
+    def tagChartMap = ["v2.7-head":"2.7.12", "v2.8-head":"2.8.3", "v2.9-head":"2.8.3"]
```

Comment:
it would be nice to have this read in from a parameter in the jenkins job, otherwise we will have to update this every release in a PR vs. having the ability to change it on the fly

Reply:
I see your point, but these will always be constants. It's going to be a PR either here or in JJB. This job is meant be run recurringly not manually or on the fly. Maybe this changes your perspective. This job is a place holder to kick off recurring runs.

Reply:
The individual jobs themselves can be updated. Doing it here, wouldn't make sense.